### PR TITLE
Update fgets.xml

### PR DESCRIPTION
**Pull request by tomasfejfar**
As described in the docs "If there is no more data to read in the file pointer, then false is returned."

### CODE REVIEW

File: reference/filesystem/functions/fgets.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
  <refsect1 role="description">
   &reftitle.description;
   <methodsynopsis>
-   <type>string</type><methodname>fgets</methodname>
+   <type>string|bool</type><methodname>fgets</methodname>
```
Comment (cmb69) [inline reference/filesystem/functions/fgets.xml:12]: Note that this should be written as:
```suggestion
   <type class="union"><type>string<type><type>bool</type></type><methodname>fgets</methodname>
```

**Review by cmb69 — COMMENTED** [review]
Thank you for the PR!

**Comment by cmb69** [comment]
Well, should actually be `string|false` not `string|bool`.

**Comment by tomasfejfar** [comment]
> Well, should actually be string|false not string|bool.

Yes, I wasn't sure if this was a supported notation in PHP manual...

**Pull request closed without merging**